advanced_functionality/multi_model_linear_learner_home_value/linear_learner_multi_model_endpoint_inf_pipeline.ipynb

```diff
@@ -0,0 +1,2859 @@
+{
```

Comment:
<p>Instead of using the S3 client, you can use the Python SDK with MultiDataModel.add_model(...): <a href="https://sagemaker.readthedocs.io/en/stable/api/inference/multi_data_model.html#sagemaker.multidatamodel.MultiDataModel.add_model" target="_blank">https://sagemaker.readthedocs.io/en/stable/api/inference/multi_data_model.html#sagemaker.multidatamodel.MultiDataModel.add_model</a></p><p> </p><p>See 'Dynamically deploying additional models' in the XGBoost MME notebook for an example: <a href="https://github.com/awslabs/amazon-sagemaker-examples/blob/master/advanced_functionality/multi_model_xgboost_home_value/xgboost_multi_model_endpoint_home_value.ipynb" target="_blank">https://github.com/awslabs/amazon-sagemaker-examples/blob/master/advanced_functionality/multi_model_xgboost_home_value/xgboost_multi_model_endpoint_home_value.ipynb</a></p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
Leaving as is, since the idea is to show that by simply copying new models to S3 location they will be part of the MME.

Reply:
All output has been cleared from the notebook.

Leaving the location of the notebook as is, to stay consistent with the other multi model endpoint examples

Reply:
MultiDataModel.add_model(...) does the same thing under the hood, but is a nice abstraction. Ultimately I think what you have is okay if we want to show the details.